arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/CssBundle.java

```diff
@@ -100,6 +100,12 @@
 
         @ClassName("doConversionBtn")
         String doConversionBtn();
+
+        @ClassName("paddL40")
+        String paddL40();

```

Comment:
Could we find a more descriptive name than paddL40 ?

Reply:
standing for **Padding Left : 40 [px]**, sounds pretty descriptive to me, doesn't it ?